paasta_tools/marathon_tools.py

```diff
@@ -185,8 +187,14 @@ def __repr__(self):
     def copy(self):
         return self.__class__(self.service, self.instance, dict(self.config_dict), dict(self.branch_dict))
 
+    def get_min_instances(self):
```

Comment:
@solarkennedy said: I'm concerned about potential confusion here. What if this was literally `min_instances` in the config, to make it very obvious which things are autoscaling and which are static?

but it got buried by a commit.